src/vue/components/word-usage-examples/word-usage-examples.vue

```diff
@@ -11,16 +11,56 @@
     </div>
 
     <div class="alpheios_word_usage_list_mainblock" v-if="showWordUsageExampleItems">
-      <div v-if="wordUsageListSorted.length > 0">
-        <word-usage-examples-item
-            v-for="wordUsageItem in wordUsageListSorted"
-            v-bind:key="wordUsageItem.ID"
-            :wordUsageItem="wordUsageItem"
-        ></word-usage-examples-item>
-      </div>
-      <div v-else>
+      <template v-if="wordUsageListSorted.length > 0">
```

Comment:
@kirlat, why did you decide to remove a separate component for the word-usage-item and place all the code inside directly word-usage-examples.vue? I have made such a structure for better independency between parts, for having smaller parts to be easier to maintain and develop.  At the end - to have less code inside one component.
What advantages do you see in decreasing an amount of components and increasing code in one component?

Reply:
A separate component as you defined it is definitely better. Unfortunately, I had to remove it due to technical limitation of a component that must have only one root element.

In order to be used in CSS grid, component must produce 5 elements (two link cells, pre and post cells, and a target word cell, see https://github.com/alpheios-project/components/issues/332#issuecomment-486750588). The simplest solution to overcome this limitation I've found is to use `v-for` without a component that can have multiple inner elements.

Please let me know if you need more info on this or if you see a better solution. Thanks!

Reply:
Oh, it is an interesting reason - I don't have enough experience with css grid.
I have read all your really rich comments in the issue - thank you for them!

I agree that it is always a best way to use  css-scalling for html layout.
And of course if a separate child component makes it not be possible, than you should move it inside the parent component.